scripts/cvmfs/setup-releases.sh

```diff
@@ -153,6 +153,25 @@ elif [ "$os" = "ubuntu22.04" ]; then
     echo "Ubuntu 22.04 detected"
 fi
 
+_replace_marlin_dll() {
+    # replace the library on MARLIN_DLL with the local one (if any)
+    local pkg_name=${1}
+    local install_prefix=${2}
+    if echo ${MARLIN_DLL} | grep -qE "/${pkg_name}/"; then
+        local old_lib=$(echo ${MARLIN_DLL} | tr ":" "\n" | grep -E "/${pkg_name}/")
+        local lib_name=$(basename ${old_lib})
+        for d in lib lib64; do
+            local new_lib=$(pwd)/${install_prefix}/${d}/${lib_name}
+            if [ -f ${new_lib} ]; then
```

Comment:
But then you have to remember to run the `k4_local_repo` after you have built. It would be fine to add the path without checking since the original is going to be removed, so it's either the one from the local build or no one

Reply:
I wasn't sure whether this is supposed to work without installing. I have removed the detection for now and simply assume that it will always be in `lib`.